drivers/flash/soc_flash_nrf.c

```diff
@@ -85,7 +85,11 @@ static inline bool is_aligned_32(u32_t data)
 	return (data & 0x3) ? false : true;
 }
 
+#ifndef CONFIG_SOC_FLASH_NRF_UICR
```

Comment:
IMO doing `#ifndef N #else  Y #endif` is confusing.  `#ifdef Y #else N #endif` is much more common.

Reply:
changes all ne #ifdef-ry to possitive wise